Dockerfile

```diff
@@ -1,14 +1,15 @@
-FROM node:14.17-alpine as build
+FROM node:16.14-alpine as build
 
 WORKDIR /server-build
 
-RUN ["apk", "add", "--no-cache", "python", "make", "gcc", "g++", "libc-dev", "sqlite-dev"]
+RUN ["apk", "add", "--no-cache", "python3", "make", "gcc", "g++", "libc-dev", "sqlite-dev"]
```

Comment:
Can we specify versions here please?

Reply:
I was doing like `RUN ["apk", "add", "--no-cache", "python3=3.9.7-r4", "make", "gcc", "g++", "libc-dev", "sqlite-dev"]
` this in local. can I revert back to `python3` ? @vraravam

Reply:
pls revert